docs/FAQ.md

```diff
@@ -219,3 +219,13 @@ To resolve the error,
    ```
 
 Details of `--signature-verification` can be found by running `man dockerd`.
+
+## Delete of replication fails with Forbidden status , although the replication gets deleted. 
+
+When customer initiates a replication delete, we create an async operation and return immediately to the customer with the operation ID to poll. This is done because DELETE replication is a long running operation and we don’t block the request until it is done. The client should continuously polls the operation for completion. The polling is done by making an GET and a sample request looks like 
```

Comment:
We don't need to include details of our internal implementation that is causing the. Instead, you can just say that replication delete from CLI requires a reader access on subscription to poll for completion and it will fail when there is no access.

Reply:
It is not internal implementation, it is general ARM asynchronous request behavior. This is common no matter where you initiate the operation CLI, Portal, Powershell.  CLI and Powershell will automatically poll for the async operation. Although they can run a command with --no-wait which will not poll in which case they don't see the issue.

Reply:
I also think it's too much implementation details. Users who come to this page most likely just want a solution or a fix.

Also there are many other scenarios than replication, essentially all async operations would fail.